### DISCUSSION TITLE
How to open multiple image paths from Command Line?

### DISCUSSION THREAD

**Discussion by gaberiel44** [d2phap/ImageGlass#1480] [Q&A]
Hello I have been trying to figure out to do this all day.

At first, I was trying this on version 8 then I gave up.

After reading this [issue](https://github.com/d2phap/ImageGlass/issues/669) I decided to upgrade to Version 9 beta 3:

Still doing something like this, in Powershell:
```
$var = "C:\temp\image_1.png", "C:\temp2\image_2.png", "C:\temp3\image_3.png"
Imageglass.exe $var
```
The application opens only with `C:\temp\image_1.png` and the other files are discarded.
On the thumbnails, only images in same folder as `C:\temp\image_1.png` are shown.

Is this not possible in V9?

Thanks!

**Comment by d2phap** [comment]
You can only open 1 path with ImageGlass, you can call it multiple times:

```
ImageGlass.exe "C:\temp\image_1.png"
ImageGlass.exe "C:\temp\image_2.png"
ImageGlass.exe "C:\temp\image_3.png"
```

ImageGlass does not support loading a custom list of image files, all images in the same folder will be added to the thumbnails list.

V9 adds support for passing settings in the `igconfig.json` as command line argument.

**Reply by gaberiel44** [reply]
I see thank you for clarifying this for me, I will make a feature request at the appropriate place
